CONTRIBUTING.md

```diff
@@ -40,10 +35,11 @@ Guidelines for bug reports:
    case](http://css-tricks.com/reduced-test-cases/) and a live example.
 
```

Comment:
live example? not sure of what it is. leave an example?

Reply:
that is not really related to what we do. We could probably re;ove the link and the live example.